### PR TITLE
Export enhancement

### PR DESCRIPTION
**Pull request by kahummer**

### CODE REVIEW

**Comment by kahummer** [comment]
@cKellyDesign The issue is specific to a ingc-moz. It addresses https://github.com/onaio/ingc-mozambique/issues/40. Is this something we want to merge to staging ?

File: src/lib/components/Export/Export.js [hunk 1]
```diff
@@ -124,7 +125,7 @@ export class Export extends Component {
     const {  mapId } = this.props;
     const { map, config } = this.state;
     const self = this;
-
+
```
Comment (cKellyDesign) [inline src/lib/components/Export/Export.js]: Remove tab

File: src/lib/components/Export/Export.js [hunk 2]
```diff
@@ -133,8 +134,11 @@ export class Export extends Component {
 
     // resize map container / map
     $(`#${mapId}.mapboxgl-map`).innerWidth(dimWidth / resValue).innerHeight(dimHeight / resValue);
-    map.removeControl(map.controls);
-    map.resize();
+    if (!config.includeNavControls) {
+      map.removeControl(map.controls);
+
```
Comment (cKellyDesign) [inline src/lib/components/Export/Export.js]: Remove line

File: src/lib/components/Export/Export.js [hunk 3]
```diff
@@ -214,8 +218,8 @@ export class Export extends Component {
       }
 
       // scale the scalling containers
-      $('.topLeft, .topRight, .bottomLeft, .bottomRight', exportEl)
-        .css('transform', `scale(${scale})`);
+      $('.topLeft, .topRight, .bottomLeft, .bottomRight', exportEl).css('transform');
+        // .css('transform', `scale(${scale})`);
```
Comment (cKellyDesign) [inline src/lib/components/Export/Export.js]: Remove comment

File: src/lib/components/Export/Export.scss [hunk 4]
```diff
@@ -203,4 +203,22 @@
...
+    background-repeat: no-repeat;
+    position: relative;
+    
+  }
+  .mapboxgl-ctrl-icon.mapboxgl-ctrl-zoom-out {
+    background-image: url("../../../../public/assets/img/zoom-out\ \(1\).png") !important;
+    background-position: 50% 50%;
+    background-repeat: no-repeat;
+    position: relative;
+  }
```
Comment (cKellyDesign) [inline src/lib/components/Export/Export.scss]: Add newline at end of file

File: src/lib/components/Export/Export.js [hunk 5]
```diff
@@ -133,8 +133,10 @@ export class Export extends Component {
 
     // resize map container / map
     $(`#${mapId}.mapboxgl-map`).innerWidth(dimWidth / resValue).innerHeight(dimHeight / resValue);
-    map.removeControl(map.controls);
-    map.resize();
+    if (!config.includeNavControls) {
+      map.removeControl(map.controls);
+    }
+      map.resize();
```
Comment (cKellyDesign) [inline src/lib/components/Export/Export.js]: Let's just fix the tabbing on this line, I'm surprised the linter didn't fix it

**Pull request merged**